www/admin/advertiser-access.php

```diff
@@ -76,7 +76,17 @@
 $oTpl->assign('unlinkPage', 'advertiser-user-unlink.php');
 
 $doUsers = OA_Dal::factoryDO('users');
-$oTpl->assign('users', array('aUsers' => $doUsers->getAccountUsersByEntity('clients', $clientid)));
+$aUsers =  $doUsers->getAccountUsersByEntity('clients', $clientid);
+$datetime_format = $date_format . " " . $time_format;
+foreach ($aUsers as $key => $aValue) {
+    // Date of last login is stored in UTC, so needs to be converted into the current user's
+    // local timezone, and then converted into the user's desired date/time format
+    $aUsers[$key]['date_last_login'] = RV_Admin_DateTimeFormat::formatUTCDateTime($aValue['date_last_login']);
+    // Date created/linked is stored in the local user's timezone at time of creation/linking,
+    // so only conversion into the user's desired date/time format is needed
+    $aUsers[$key]['date_created'] = RV_Admin_DateTimeFormat::formatDateTime($aValue['date_created']);
```

Comment:
I suppose it was the "local tz" of the user that created the account at the time of creation. Which could not be the TZ of the user currently watching the user list.

I think our best bet is to store the date as UTC and make the regular UTC-local conversion here. Also, we could:
* try to migrate the old values or
* display a warning similar to what we do with stats if we detect an old date, or
* do nothing, as it's not super valuable to be accurate here

Reply:
Correct, it appears that we use the local TZ of the user creating the account which is used when storing the created date. This is not ideal, but as discussed in #1051, migrating this field to UTC is technically impossible, because we can't know what that local TZ was. 

Personally, I was going to just ignore this issue for this ticket, because as you say, it's not important to be super accurate here. I know @erikgeurts has suggested that we just remove the column; but I think there is some value in keeping the column present.

In fact, now that I think about it, given that we can't be completely accurate, why don't we just display the date of creation, rather than date/time? It's possible it may not be exactly the right date, if created near midnight, but using a date only does imply some degree of lack of accuracy, I feel.... And the column is called "Date linked" after all!

Committing.

Reply:
I still think we should change the creation so that UTC is used, like everywhere else, without leaving this field as an odd one.
The "Date linked" won't be accurate for older accounts, but will always be for new installs and new accounts on old instances. If you feel it's not worth it, merge away ;-)